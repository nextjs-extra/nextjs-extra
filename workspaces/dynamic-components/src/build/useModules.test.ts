import { useModules } from "./useModules";

test("useModules", () => {
  const modules = ["eslint", "react/jsx-runtime", "this-module-does-not-exist"];

  expect(useModules(modules)).toMatchSnapshot();
});
