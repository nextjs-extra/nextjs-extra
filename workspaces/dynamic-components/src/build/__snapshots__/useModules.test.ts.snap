// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useModules 1`] = `
"
  eslint: dynamic(() => import("eslint"), { suspense: true }),
  reactãƒ¼jsx-runtime: dynamic(() => import("react/jsx-runtime"), { suspense: true }),"
`;
